cmd/adsysd/integration_tests/adsys_mount_test.go

```diff
@@ -41,7 +43,7 @@ func TestUserMountHandler(t *testing.T) {
 
 		// File errors
 		"error when file has badly formated entries": {mountsFile: "mounts_with_bad_entries", wantStatus: 1},
-		"error when file doesn't exist":              {mountsFile: "do_not_exist", wantStatus: 1},
+		"error when file doesn't exist":              {mountsFile: "do_not_exist", wantStatus: 2},
```

Comment:
This should be 1, it’s not a syntax issue.

Reply:
Should it? The os error code returned by Rust is 2.

Reply:
Oh, status 2 is reserved to misuse of shell builtins

Reply:
It should be 1, as it’s an internal process error. Unsure how Rust decide about exit process on internal error.

Reply:
From what I understood, everything that errors out of your code (i.e. an error from a main function), always counts as status 1. That's why I had to change the way we returned out of the main function in order to be able to get a status 2 when we have invalid args.